apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -49,22 +81,443 @@ export const getPageInfo: GetPageNavInfo = ({
+          })}
+        >
+          {screeningQuestionResponses.length > 0 ? (
+            <div>
+              <p data-h2-margin-bottom="base(x1)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "You’ve answered the following screening questions:",
+                  id: "GJc/j5",
+                })}
+              </p>
+              <ul>
+                {screeningQuestionResponses.map((response) => (
+                  <li key={response.id} data-h2-margin-bottom="base(x1)">
+                    <p
+                      data-h2-font-weight="base(700)"
+                      data-h2-margin-bottom="base(x.5)"
+                    >
+                      {response.screeningQuestion?.question
+                        ? response.screeningQuestion.question[locale]
+                        : ""}
+                    </p>
+                    <p>{response.answer}</p>
+                  </li>
+                ))}
+              </ul>
+            </div>
+          ) : (
+            <Well>
+              <p data-h2-text-align="base(center)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "It looks like you haven't answered any screening questions yet.",
+                  id: "+ojoEd",
+                })}
+              </p>
+            </Well>
+          )}
+        </ReviewSection>
+      )}
+
+      <section data-h2-margin-bottom="base(x3)">
+        <Heading level="h3" data-h2-margin="base(0, 0, x1, 0)">
+          {intl.formatMessage({
+            defaultMessage: "Sign and submit",
+            id: "+sWqVY",
+          })}
+        </Heading>
+        <div>
+          <FormProvider {...methods}>
+            <form onSubmit={methods.handleSubmit(handleSubmit)}>
+              <p data-h2-margin-bottom="base(x1)">
+                {intl.formatMessage({
+                  defaultMessage: `"You made it! By signing your name, you confirm that:"`,
+                  id: "d0KeTK",
+                })}
+              </p>
+              <ul>
+                <li>
+                  <p data-h2-margin-bottom="base(x.5)">
+                    {intl.formatMessage({
+                      defaultMessage: `"I’ve reviewed everything written in my application"`,
+                      id: "IFVdRD",
+                    })}
+                  </p>
+                </li>
+                <li>
+                  <p data-h2-margin-bottom="base(x.5)">
+                    {intl.formatMessage({
+                      defaultMessage: `"I understand that I am part of a community who trusts each other"`,
+                      id: "MKU7g2",
+                    })}
+                  </p>
+                </li>
+                <li>
+                  <p data-h2-margin-bottom="base(x.5)">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "I promise that the information I’ve provided is true",
```

Comment:
Shouldn't this string be between quotes if the other two statements are?

Reply:
Nice catch, fixed by [04cd097](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/04cd097ac0488a7c7b73635a8d52ea454f945b16)